templates/about.html

```diff
@@ -1,188 +1,81 @@
+                        data from over 100 exchanges and serve them from a single URL or using Tor as an onion hidden
+                        service. It's extremely flexible: the user can decide which base currencies to use (USDT, USD,
+                        EUR etc), which exchanges to keep data for, and how much data to keep
+                        <div>Learn more on how you can setup your own Spotbit on<a
+                            href="https://github.com/BlockchainCommons/spotbit"> github</a>.
+                        </div>
+
+                      </div>
 
-                          </div>
-        
                     </div>
+                  </div>
+                  </p>
+
                 </div>
-            </p>
-            
-          </div>
-          <div class="crypto-widget__content">
-            <p class="mdc-typography--body1">
                 <div class="crypto-widget__content">
+                  <p class="mdc-typography--body1">
+                  <div class="crypto-widget__content">
                     <div class="crypto-wallet-overview">
-                           <div>Spotbit can be used by anyone who wants to take advantage of its privacy, speed, reliability, and self-sovereignty advantages. It works particularly well with the <a href="https://github.com/sponsors/BlockchainCommons">Gordian system</a>, which supports Tor connections between a <a href="https://github.com/sponsors/BlockchainCommons">Gordian Wallet</a> and a <a href="https://github.com/sponsors/BlockchainCommons"></a>Gordian Server or other server installed by <a href="https://github.com/sponsors/BlockchainCommons">Bitcoin Standup scripts</a>.
-                            Spotbit is an example of a microservices meant to display how the <a href="https://github.com/BlockchainCommons/spotbit">Gordian Principles</a>, which are philosophical and technical underpinnings to Blockchain Commons' Gordian technology, are supported by the Gordian Architecture. 
+                      <div>Spotbit can be used by anyone who wants to take advantage of its privacy, speed, reliability,
+                        and self-sovereignty advantages. It works particularly well with the <a
+                          href="https://github.com/sponsors/BlockchainCommons">Gordian system</a>, which supports Tor
+                        connections between a <a href="https://github.com/sponsors/BlockchainCommons">Gordian Wallet</a>
+                        and a <a href="https://github.com/sponsors/BlockchainCommons"></a>Gordian Server or other server
+                        installed by <a href="https://github.com/sponsors/BlockchainCommons">Bitcoin Standup
+                          scripts</a>.
+                        Spotbit is an example of a microservices meant to display how the <a
+                          href="https://github.com/BlockchainCommons/spotbit">Gordian Principles</a>, which are
```

Comment:
Can we use the correct links:
- [Gordian System](https://github.com/BlockchainCommons/Gordian)
- [Gordian Principles](https://github.com/BlockchainCommons/Gordian#gordian-principles)
- [Gordian Wallet](https://github.com/BlockchainCommons/GordianWallet-iOS)
- [Bitcoin Standup Scripts](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts)